plugins/woocommerce/src/Admin/Settings/SettingsSection.php

```
@@ -0,0 +1,38 @@
+ *
+ * @since 10.9.0
+ */
+abstract class SettingsSection implements SettingsSectionInterface {
+
+	/**
+	 * Get script handles that must be loaded before the settings UI app mounts.
+	 *
+	 * @param \WC_Settings_Page $parent_page Parent settings page.
+	 * @return string[]
+	 */
+	public function get_script_handles( \WC_Settings_Page $parent_page ): array {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$parent_page'. (undefined)

(UnusedFormalParameter)